prometheus_api_client/metric.py

```diff
@@ -155,9 +155,10 @@ def __add__(self, other):
         """
         if self == other:
             new_metric = deepcopy(self)
-            new_metric.metric_values = new_metric.metric_values.append(
-                other.metric_values, ignore_index=True
-            )
+            new_metric.metric_values = pandas.concat([new_metric.metric_values,other.metric_values],ignore_index=True)
```

Comment:
Small nit: would you mind explicitly passing the value for `axis` parameter in concat (I believe it should be `axis=0`), just to be extra cautious about how the values should be concatenated?

Reply:
@chauhankaranraj added `axis=0`. Thanks for the suggestion.

Reply:
Great, thanks!